bitnami/pgbouncer/README.md

```diff
@@ -115,6 +115,7 @@ docker run --name pgbouncer \
 
 In case you'd like pgbouncer to expose your database with a different name, you can use the `PGBOUNCER_DATABASE` variable.
 To expose the same database name as the backend, set `PGBOUNCER_DATABASE="$POSTGRESQL_DATABASE"`.
+To expose a ["fallback database" (wildcard that matches any)](https://www.pgbouncer.org/config.html#:~:text=%E2%80%9C*%E2%80%9D%20acts%20as%20a%20fallback%20database%3A%20If%20the%20exact%20name%20does%20not%20exist%2C%20its%20value%20is%20taken%20as%20connection%20string%20for%20the%20requested%20database.%20For%20example%2C%20if%20there%20is%20an%20entry%20(and%20no%20other%20overriding%20entries)), set `PGBOUNCER_DATABASE="*"`.
```

Comment:
I'm sorry for the late response, but could you please replace the link with 'https://www.pgbouncer.org/config.html#section-databases' ?

```suggestion
To expose a ["fallback database" (wildcard that matches any)](https://www.pgbouncer.org/config.html#section-databases)), set `PGBOUNCER_DATABASE="*"`.
```

Reply:
Of course. I would have preferred to link to a headline for the `*` section of this paragraph, but as it does not exist, maybe your choice is better.